src/main/resources/login.properties

```diff
@@ -0,0 +1,2 @@
+password=hqhTqwje872H
```

Comment:
Why don't you create a file for all the test data? currently you can not store anything else here. Make it more generic

Reply:
done